src/js/message.js

```diff
@@ -95,6 +119,7 @@ class Message {
 
 	/**
 	 * Open the message.
+	 * @returns {undefined}
```

Comment:
This function does not return an explicit undefined, which is what this return documentation is stating.

To document an implicit undefined you need to use `void`, according to what my editor is telling me anyway

```suggestion
	 * @returns {void}
```

Reply:
cool thanks, done :) I had a quick look at the `@returns` docs and couldn't see anything for implicit undefined returns